Remotion/Validation/Globalization.UnitTests/LocalizedValidationMessageFactoryTest.cs

```diff
@@ -280,6 +280,32 @@ public void CreateValidationMessageForPropertyValidator_WithNotEmptyCollectionVa
       Assert.That(validationMessage.ToString(), Is.EqualTo("Add an entry."));
     }
 
+    [Test]
+    public void CreateValidationMessageForPropertyValidator_WithNotEmptyOrWhitespaceValidatorAndPropertyIsString_ReturnsLocalizedValidationMessage ()
```

Comment:
please recheck the tests, I think they are red now because they check the English localization.

Reply:
Amended tests; all tests are now green.